PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1821,24 +1824,34 @@ void FindReplaceDlg::findAllIn(InWhat op)
 		cmdid = WM_FINDINFILES;
 	else if (op == CURRENT_DOC)
 		cmdid = WM_FINDALL_INCURRENTDOC;
-
-	if (!cmdid) return;
+	else
+		return;
 
 	if (::SendMessage(_hParent, cmdid, 0, 0))
 	{
-		if(_findAllResult == 1)
-			wsprintf(_findAllResultStr, TEXT("1 hit"));
-		else
-			wsprintf(_findAllResultStr, TEXT("%d hits"), _findAllResult);
-		if (_findAllResult) 
+		if (_findAllResult > 0) 
 		{
+			if (_findAllResult == 1)
+				wsprintf(_findAllResultStr, TEXT("1 hit"));

```

Comment:
In `PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp` there are numerous other places where `wsprintf` is used actually. It's better for clarity sake to make a new patch that replaces all the insecure api function calls in the file with its security counterpart.
I'll do it.